### DISCUSSION TITLE
GitHub Student Pack Approved but Benefits Not Received After 5 Days – Need Assistance

### DISCUSSION THREAD

**Discussion by FATTAL-FairFight** [community/community#144687] [GitHub Education]
### Hi everyone,

Application Status: Approved
Approval Date: 12/Nov/2024
Steps Taken So Far:
Checked my email (including spam/junk folders)
Reviewed GitHub’s [Student Pack FAQs](https://education.github.com/pack)

Do I need to wait for a week?

**Comment by davevad93** [comment]
Hi @FATTAL-FairFight ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @FATTAL-FairFight was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
